pkg/cloudprovider/fake/controller-kwok/charts/README.md

```diff
@@ -0,0 +1,88 @@
+# karpenter
+
+A Helm chart for Karpenter, an open-source node provisioning project built for Kubernetes.
+
+![Version: 0.33.0](https://img.shields.io/badge/Version-0.33.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.33.0](https://img.shields.io/badge/AppVersion-0.33.0-informational?style=flat-square)
+
+## Documentation
+
+For full Karpenter documentation please checkout [https://karpenter.sh](https://karpenter.sh/docs/).
+
+## Installing the Chart
+
+You can follow the detailed installation instruction in the [documentation](https://karpenter.sh/docs/getting-started/getting-started-with-karpenter) which covers the Karpenter prerequisites and installation options. The outcome of these instructions should result in something like the following command.
```

Comment:
Should we reference the AWS docs for this right now?

Reply:
I just copied this from the charts in the AWS repo. I think the AWS getting started guide comes with a lot more than necessary for this, what are you thinking is not enough/too much to maintain with this?